backend/coreapp/tests/test_compilation.py

```diff
@@ -0,0 +1,257 @@
+from coreapp import compilers
+from coreapp.compiler_wrapper import CompilerWrapper
+from coreapp.compilers import (
+    GCC281PM,
+    IDO53,
+    IDO71,
+    MWCC_247_92,
+    PBX_GCC3,
+    WATCOM_105_C,
+    Compiler,
+    DummyCompiler,
+)
+from coreapp.diff_wrapper import DiffWrapper
+from coreapp.flags import Language
+from coreapp.models.scratch import Assembly
+from coreapp.platforms import N64
+from coreapp.tests.common import BaseTestCase, requiresCompiler
+from django.urls import reverse
+from parameterized import parameterized
+from rest_framework import status
+
+
+class CompilationTests(BaseTestCase):
```

Comment:
should we add a 'create-scratch-from-preset' test?

Reply:
it's in test_preset.py